.github/workflows/firebase-hosting-merge.yml

```diff
@@ -15,11 +12,12 @@ jobs:
           fetch-depth: 0
       - name: Generate Site
         run: make site-generate
-      - uses: FirebaseExtended/action-hosting-deploy@v0
+      - uses: FirebaseExtended/action-hosting-deploy@276388dd6c2cde23455b30293105cc866c22282d # v0
```

Comment:
Why using hash instead of v0?

Reply:
Rationale at https://github.com/GoogleContainerTools/kpt/pull/1515#discussion_r586901305

Reply:
FirebaseExtended is not authored by the team in Google?

Reply:
According to https://github.com/FirebaseExtended, it's "not officially staffed by Google"